llama-index-integrations/llms/llama-index-llms-bedrock/llama_index/llms/bedrock/base.py

```diff
@@ -198,6 +199,8 @@ def _model_kwargs(self) -> Dict[str, Any]:
             "temperature": self.temperature,
             self._provider.max_tokens_key: self.max_tokens,
         }
+        if type(self._provider) is AnthropicProvider:
```

Comment:
```suggestion
        if type(self._provider) is AnthropicProvider and self.system_prompt:
```

For dropping null values